### ISSUE TITLE
Get the APK of any package on the phone

### ISSUE THREAD

**Issue by 0x192** [0x192/universal-android-debloater#118]
A lot of apps cannot be found on the Web. This prevents contributors to help documenting apps they don't have. 

I often ask users to upload the APK manually but we could add this feature directly in UAD so that you only have to press a button to pull the APK.

**Comment by lawson85** [comment]
A play store link for available packages would be good i think

**Comment by KarlRamstedt** [comment]
> A play store link for available packages would be good i think

Yeah, generating a link is literally as simple as appending the package name to `https://play.google.com/store/apps/details?id=`

Most packages aren't on the Play Store tho, so would be nice if UAD could check for that? That would be a bit harder to code tho, and might slow down loading significantly (since it has to check a web-page for every single package).

**Comment by lawson85** [comment]
If it returns a 404 then it doesn't exist so maybe that's a good starting 
point. 200 means it worked
